metrics/prometheus.go

```diff
@@ -1585,7 +1589,13 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 	return c
 }
 
-var versionInfoDesc = prometheus.NewDesc("cadvisor_version_info", "A metric with a constant '1' value labeled by kernel version, OS version, docker version, cadvisor version & cadvisor revision.", []string{"kernelVersion", "osVersion", "dockerVersion", "cadvisorVersion", "cadvisorRevision"}, nil)
+var (
+	versionInfoDesc = prometheus.NewDesc("cadvisor_version_info", "A metric with a constant '1' value labeled by kernel version, OS version, docker version, cadvisor version & cadvisor revision.", []string{"kernelVersion", "osVersion", "dockerVersion", "cadvisorVersion", "cadvisorRevision"}, nil)
+	startTimeDesc   = prometheus.NewDesc("container_start_time_seconds", "Start time of the container since unix epoch in seconds.", nil, nil)
```

Comment:
Is this just a bug-fix?  Did we not have these before?

Reply:
Yes. Prometheus client was complaining about unregistered desriptions - you can see same descriptions (plus labels) [below](https://github.com/iwankgb/cadvisor/blob/bc4596f2446b185c700d34861b4394bebc38ce33/metrics/prometheus.go#L1715).